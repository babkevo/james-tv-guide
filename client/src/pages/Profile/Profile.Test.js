import React, { Component } from "react";
import renderer from 'react-test-renderer';
import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";

test('Link to Profile test', () => {
    const component = renderer.create(
    <Router>  <Link to="localhost:3000/tvlist/alltv">All Program list </Link> </Router> 
    );

    let testcategories = component.toJSON();
    expect(testcategories).toMatchSnapshot();
  


});