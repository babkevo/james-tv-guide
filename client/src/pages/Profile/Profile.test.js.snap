// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link to Profile test 1`] = `
Array [
  "  ",
  <a
    href="/localhost:3000/tvlist/alltv"
    onClick={[Function]}
  >
   All Program list
  </a>,
  " ",
]
`;
